modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -21,6 +21,66 @@ definition {
 		}
 	}
 
+	@description = "LPS-150886- Verify that it's possible to delete an object after deleting the relationship"
+	@priority = "4"
+	test CanDeleteCustomObjectAfterDeleteRelationship {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 290",
+			objectName = "CustomObject290",
+			pluralLabelName = "Custom Objects 290");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldBusinessType = "Text",
+			fieldLabelName = "Custom Field",
+			fieldName = "customObjectField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject290");
+
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			objectName_1 = "CustomObject290",
+			objectName_2 = "User",
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship",
+			relationshipType = "oneToMany");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject290");
+
+		ObjectAdmin.openObjectAdmin();
+
+		Click(
+			key_objectLabel = "Custom Object 290",
+			locator1 = "ObjectAdmin#OBJECT_KEBAB_MENU");
+
+		MenuItem.clickNoError(menuItem = "Delete");
+
+		SelectFrame(locator1 = "ObjectCustomViews#VIEW_MODAL_CONTENT");
+
+		ObjectAdmin.assertCannotDeleteObjectWithRelationshipErrorMessage();
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectSystemObject(label = "Custom Object 290");
+
+		ObjectAdmin.gotoRelationshipsTab();
+
+		ObjectAdmin.deleteRelationshipViaUI(
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship");
+
+		ObjectAdmin.assertObjectRelationshipNotPresent(relationshipLabel = "Relationship");
+
```

Comment:
We don't need to assert the relationship is deleted here since that's not within the scope of the test. We generally want to avoid adding assertions that are covered in other tests and keep the scope narrow, to avoid redundant testing but also to prevent our tests from becoming fragile. If multiple tests are making the same assertion, it means all of them will break if a regression is introduced.